packages/firestore/CHANGELOG.md

```diff
@@ -1,4 +1,14 @@
 # Unreleased
+- [breaking] `QuerySnapshot.docChanges()` is now a method. Please make sure
+  to audit your code for all existing usage of the old property `.docChanges`
+  and to change these to method invocations.
+- [changed] Replaced `QueryListenOptions` and `DocumentListenOptions` with
+  `SnapshotListenOptions`. `SnapshotListenOptions` only exposes the single
+  property `includeMetadataChanges` to turn on metadata changes.
```

Comment:
Perhaps don't both mentioning these types since I doubt anybody is explicitly referencing them.  Instead reference the option names...  Something like:

Merged the `includeQueryMetadataChanges` and `includeDocumentMetadataChanges` options passed to `Query.onSnapshot()` into a single `includeMetadataChanges` option.

[I don't think we need to mention DocumentReference.onSnapshot() at all since the usage didn't change]

Reply:
Sounds good! I like this much better.